custom/ilsgateway/tanzania/handlers/keyword.py

```diff
@@ -27,6 +27,9 @@ def sql_location(self):
         if location:
             return location.sql_location
 
+        if not self.args:
+            return
```

Comment:
Yes and it's correct because user needs to have location assigned or send site_code as a first parameter in soh message.